dist/shiny-server-client.js

```diff
@@ -1764,6 +1759,10 @@ exports.init = function (shinyServer, disableProtocols) {
 
   if (whitelist.length == 0) {
     whitelist = availableOptions;
+    // MS Edge works very poorly with xhr-streaming (repro'd with shinyapps.io and RSC on Edge 17.17134)
+    if (/\bEdge\//.test(window.navigator.userAgent)) {
+      whitelist.splice($.inArray("xhr-streaming", whitelist), 1);
+    }
```

Comment:
I think this actually needs to be above, under the `if (!whitelistStr || whitelistStr === "")` clause. Currently it'd only be hit if HTML5 storage is not supported, or if there was a persisted setting that didn't contain any (valid) protocols, right?